cli/src/semgrep/target_manager.py

```diff
@@ -106,7 +106,7 @@ class FileTargetingLog:
     cli_includes: Set[Path] = Factory(set)
     cli_excludes: Set[Path] = Factory(set)
     size_limit: Set[Path] = Factory(set)
-    failed_to_analyze: Set[Path] = Factory(set)
+    failed_to_analyze: Mapping[Path, Optional[int]] = Factory(dict)
```

Comment:
Can we add comment on what this is mapping / make the variable name more clear what it is mapping file to?